### ISSUE TITLE
Could you tell me the kinetics's details in accuracy of you project

### ISSUE THREAD

**Issue by Violetgyw** [v-iashin/video_features#3]

**Comment by v-iashin** [comment]
I haven't evaluated the pre-trained models on Kinetics. This repo wraps up the original pre-trained checkpoints for feature extraction. Therefore, the quality should be the same as of the original models. 

In case of R(2+1)D RGB-only, it should match the reported one in [here](https://pytorch.org/docs/1.5.0/torchvision/models.html#video-classification). One thing, however, it would be difficult to obtain the same quality of features for I3D because in the original paper, they use TLV1 optical flow. In this repo, there are two faster options to calculate the optical flow: PWC and RAFT.

I would be nice if someone could evaluate the quality of the methods from this repo by training the classification layer, esp I3D with different flow extractors.

**Comment by v-iashin** [comment]
By the way, another option on how to evaluate the performance would be to use `--show_pred` argument [from here](https://github.com/v-iashin/video_features/blob/b11c2a36c0ae882c65b3d083e340244c56fa803a/main.py#L81) which shows the predictions on the kinetics (classes and probs). But one still needs to change the script and download the dataset

**Comment by Violetgyw** [comment]
> By the way, another option on how to evaluate the performance would be to use `--show_pred` argument [from here](https://github.com/v-iashin/video_features/blob/b11c2a36c0ae882c65b3d083e340244c56fa803a/main.py#L81) which shows the predictions on the kinetics (classes and probs). But one still needs to change the script and download the dataset

Thank you for your answer, I tried to use TV-L1, RAFT, PWC, the sample v_CricketShot_g04_c01.mp4 provided at https://github.com/deepmind/kinetics-i3d, using the same processing method, I found only TV-L1 Which category can be accurately distinguished, the official i3d uses TV-L1, other optical flow distributions are different, the effect is not good as it should be

**Comment by v-iashin** [comment]
Can you provide a code snippet? I am afraid, I have a hard time understanding what you mean.

**Comment by varungupta31** [comment]
@Violetgyw Please share which repo you used for I3D with TV-L1?
